changelogs/unreleased/db-migration-test-fixture.yml

```diff
@@ -0,0 +1,5 @@
+description: Added a generic fixture for db migration tests that can be used by core and extensions.
+change-type: patch
+destination-branches:
+  - master
+  - iso4
```

Comment:
Should I merge this into iso3 as well? It might give issues with the `FixtureRequest`, like we had on master last week. On the other hand, if we don't merge it it might make db patches for all three branches more difficult because iso3 then requires some additional work.

Reply:
Iso3 is almost off the board, so whatever is least effort